mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/MessageParserTest.java

```diff
@@ -201,9 +201,23 @@ public void getAttachmentsShouldNotConsiderUnknownContentDispositionAsAttachment
 
     @Test
     public void getAttachmentsShouldRetrieveAttachmentsWhenNoCidForInlined() throws Exception {
-        List<MessageAttachment> attachments = testee.retrieveAttachments(ClassLoader.getSystemResourceAsStream("eml/multiAttachmentsWithOneWrongInlinedAttachment.eml"));
```

Comment:
You did not remove `multiAttachmentsWithOneWrongInlinedAttachment` is it used by any other test ? If no please remove it